docs/client/java/partials/java_transport.md

```diff
@@ -141,99 +257,224 @@ transport:
-| openlineage.transport.roleArn           | Optional, the roleArn which is allowed to read/write to Kinesis stream                                                                                                                       | some-role-arn    |
-| openlineage.transport.properties.[xxx]  | Optional, the [xxx] is property of [Kinesis allowd properties](https://github.com/awslabs/amazon-kinesis-producer/blob/master/java/amazon-kinesis-producer-sample/default_config.properties) | 1                |
-
+```ini
+openlineage.transport.type=kinesis
+openlineage.transport.streamName=your_kinesis_stream_name
+openlineage.transport.region=your_aws_region
+openlineage.transport.roleArn=arn:aws:iam::account-id:role/role-name
+openlineage.transport.properties.VerifyCertificate=true
+openlineage.transport.properties.ConnectTimeout=6000
+```
 
 </TabItem>
-</Tabs>
+<TabItem value="java" label="Java Code">
 
-*Behavior*:
-- Events are serialized to JSON upon the `emit()` call and dispatched to the Kinesis stream.
-- The partition key is generated by combining the job's namespace and name.
-- Two constructors are available: one accepting both `KinesisProducer` and `KinesisConfig` and another solely accepting `KinesisConfig`.
+```java
+import java.util.Properties;
+
+import io.openlineage.client.OpenLineageClient;
+import io.openlineage.client.transports.KinesisConfig;
+import io.openlineage.client.transports.KinesisTransport;
 
+Properties kinesisProperties = new Properties();
+kinesisProperties.setProperty("property_name_1", "value_1");
+kinesisProperties.setProperty("property_name_2", "value_2");
 
-### [Console](https://github.com/OpenLineage/OpenLineage/tree/main/client/java/src/main/java/io/openlineage/client/transports/ConsoleTransport.java)
+KinesisConfig kinesisConfig = new KinesisConfig();
+kinesisConfig.setStreamName("your_kinesis_stream_name");
+kinesisConfig.setRegion("your_aws_region");
+kinesisConfig.setRoleArn("arn:aws:iam::account-id:role/role-name");
+kinesisConfig.setProperties(kinesisProperties);
+
+OpenLineageClient client = OpenLineageClient.builder()
+  .transport(
+    new KinesisTransport(httpConfig))
+  .build();
+```
+
+</TabItem>
+</Tabs>
+
+#### [Console](https://github.com/OpenLineage/OpenLineage/tree/main/client/java/src/main/java/io/openlineage/client/transports/ConsoleTransport.java)
 
 This straightforward transport emits OpenLineage events directly to the console through a logger.
-Be cautious when using the DEBUG log level, as it might result in double-logging due to the `OpenLineageClient` also logging.
 No additional configuration is required.
 
+##### Behavior
+
+Events are serialized to JSON. Then each event is logged with `INFO` level to logger with name `ConsoleTransport`.
+
+##### Notes
+
+Be cautious when using the `DEBUG` log level, as it might result in double-logging due to the `OpenLineageClient` also logging.
+
```

Comment:
Could you please also add that to Python console client? It works in the same way.

Reply:
Done